packages/core-modules/contracts/modules/NftModule.sol

```diff
@@ -11,7 +11,7 @@ import "../storage/NftStorage.sol";
 
 import "../interfaces/INftModule.sol";
 
-contract NftModule is INftModule, ERC721, NftStorage, InitializableMixin, OwnableMixin {
+contract NftModule is INftModule, ERC721Enumerable, NftStorage, InitializableMixin, OwnableMixin {
```

Comment:
ooh this is beyond a doc change, its a code change. is this intentional? I can agree it the change in general.